java/src/client/game/GameManager.java

```diff
@@ -25,20 +26,41 @@
     /**
      * The server that may be a fake one or a real one
      */
-    IServer mServer;
+    private IServer mServer;
+    /**
+     * Handel to a facade for the current game, allows for use on both client and server - with n games.
```

Comment:
im hoping you have more productive things to do with your time than correct minor spelling errors on comments where no meaning is lost.....